### PR TITLE
Update walkthrough to match product features

### PR DESCRIPTION
**Pull request by taylorsperry**
Update in-editor onboarding walkthrough. Adds "Ask Cody to Fix" and "Natural Language Search" to the walkthrough and updates gifs to reflect new UI. 

## Test plan

- Opened walkthrough
- Clicked through the guides

### CODE REVIEW

File: vscode/walkthroughs/autocomplete.md [hunk 1]
```diff
@@ -1,10 +1,5 @@
 ## Code Autocomplete
 
-<video autoPlay muted loop playsInline>
-    <source
-        type="video/mp4"
-        src="https://storage.googleapis.com/sourcegraph-assets/website/Product%20Animations/cody-completion-may2023.mp4"
-    />
-</video>
+//TODO: Add new autocomplete gif
```
Comment (taylorsperry) [inline vscode/walkthroughs/autocomplete.md:3]: @jdorfman is working on this!

File: vscode/walkthroughs/fix.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Ask Cody to Fix
+
+//TODO: Add Fix gif
```
Comment (taylorsperry) [inline vscode/walkthroughs/fix.md:3]: @jdorfman is working on this!

File: vscode/walkthroughs/search.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+## Natural Language Search
+
+//TODO: Add NLS gif
```
Comment (taylorsperry) [inline vscode/walkthroughs/search.md:3]: @jdorfman is working on this!

File: vscode/package.json [hunk 4]
```diff
@@ -141,7 +149,7 @@
           {
             "id": "learn-more",
             "title": "Learn More & Feedback",
-            "description": "📖 Dive deeper into Cody by reading our [full documentation](https://docs.sourcegraph.com/cody).\n🎨 Discover more features by searching for \"Cody\" in the [Command Palette](command:workbench.action.showCommands).\n🗒️ Find out how Cody is improving by taking a look at the [Changelog](https://sourcegraph.com/github.com/sourcegraph/cody/-/blob/vscode/CHANGELOG.md).\n💬 Bugs, ideas or feedback? Post a message on our [GitHub Discussions](https://github.com/sourcegraph/cody/discussions/new?category=product-feedback&labels=vscode).",
+            "description": "📖 Dive deeper into Cody by reading our [full documentation](https://docs.sourcegraph.com/cody).\n🎨 Discover more features by searching for \"Cody\" in the [Command Palette](command:workbench.action.showCommands).\n🗒️ Find out how Cody is improving by taking a look at the [Changelog](https://sourcegraph.com/github.com/sourcegraph/cody/-/blob/vscode/CHANGELOG.md).\n💬 Bugs, ideas or feedback? Raise a [GitHub Issue](https://github.com/sourcegraph/cody/issues/new/choose).",
```
Comment (taylorsperry) [inline vscode/package.json:152]: @kalanchan will this docs link (https://docs.sourcegraph.com/cody) still work with the new docs?

File: vscode/walkthroughs/commands.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 ## Cody Commands
 
-Cody can generate unit tests, documentation, and more, leveraging its full awareness of your codebase. You can access and create various helpful commands to streamline your development process with just a few clicks.
+<img src="https://storage.googleapis.com/sourcegraph-assets/blog/vs-code-onboarding-walkthrough-dec-2023-explain.gif">
 
-To get started, select a command from the [Cody: Commands Menu](command:cody.action.commands.menu), or right click on a code section.
+Cody can explain code, generate unit tests, add documentation, and more, by leveraging its full awareness of your codebase. You can also build your own [Custom Commands (Beta)](https://docs.sourcegraph.com/cody/capabilities/commands#custom-commands) to tailor Cody to your workflow.
```
Comment (taylorsperry) [inline vscode/walkthroughs/commands.md:5]: @kalanchan will this docs link (https://docs.sourcegraph.com/cody/capabilities/commands#custom-commands) still work with the new docs?

File: vscode/walkthroughs/explain.md [hunk 6]
```diff
@@ -1,12 +1,14 @@
 ## Explain Selected Code
 
+//TODO: Remove from package.json
```
Comment (kalanchan) [inline vscode/walkthroughs/explain.md:3]:
```suggestion

```

I think you removed the above in package.json already, cleaning up comments

**Review by kalanchan — APPROVED** [review]
lgtm!

**Comment by toolmantim** [comment]
I've added a whole bunch of pro-tips for each section, with keyboard shortcuts, ways to customise it, get the most out of it, etc. I also added some buttons to the items.

I had to use a weird unordered list format to work around VS Code’s awkward CSS styles.

**Review by toolmantim — APPROVED** [review]

**Pull request merged**